mavros_extras/src/plugins/esc_status.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @brief ESC status plugin
+ * @file esc_status.cpp
+ * @author Ricardo Marques <marques.ricardo17@gmail.com>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2020 Ricardo Marques <marques.ricardo17@gmail.com>.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros_plugin.h>
+#include <mavros_msgs/ESCInfo.h>
+#include <mavros_msgs/ESCStatus.h>
+
+namespace mavros
+{
+namespace extra_plugins
+{
+/**
+ * @brief ESC status plugin
+ */
+class ESCStatusPlugin : public plugin::PluginBase
+{
+public:
+	ESCStatusPlugin() : PluginBase(),
+		nh("~")
```

Comment:
Please add _max_esc_info_index & _max_esc_status_index to initializer list.
It's actually not necessary, but i think better to have more explicit code.

Reply:
Sounds good.